scripts/backCompatChecker.py

```diff
@@ -81,8 +85,9 @@ def findExtension():
 # get a set of old commands, we are not reusing the set from ext because we want to keep this clean
 oldCommands = []
 for oldArgument in oldArguments:
-    if not any(oldArgument.command in s for s in oldCommands):
-        oldCommands.append(oldArgument.command)
+    if oldArgument.command not in ignoreCommands:
+        if not any(oldArgument.command in s for s in oldCommands):
+            oldCommands.append(oldArgument.command)
 
```

Comment:
Exclude this from 
https://github.com/Microsoft/azure-devops-cli-extension/blob/93cae2800ffeb74e55fafbf74777bc3ed8a6913a/scripts/backCompatChecker.py#L87-L90
as well

Reply:
This was not required. 
There were other issues which I fixed.